ui/raidboss/data/06-ew/raid/p4s.ts

```diff
@@ -0,0 +1,704 @@
+      netRegexJa: NetRegexes.startsUsing({ id: '69D7', source: 'ヘスペロス', capture: false }),
+      alertText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Go to Corner',
+          de: 'In eine Ecke gehen',
+          fr: 'Allez dans un coin',
+          cn: '去角落',
+          ko: '구석으로',
+        },
+      },
+    },
+    {
+      id: 'P4S Well Pinax',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '69D6', source: 'Hesperos', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ id: '69D6', source: 'Hesperos', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ id: '69D6', source: 'Hespéros', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ id: '69D6', source: 'ヘスペロス', capture: false }),
+      delaySeconds: 5,
+      alertText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Middle Knockback',
+          de: 'Rückstoß von der Mitte',
+          fr: 'Poussée au milieu',
+          cn: '中间击退',
+          ko: '중앙에서 넉백',
+        },
+      },
+    },
+    {
+      id: 'P4S Acid Pinax',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '69D4', source: 'Hesperos', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ id: '69D4', source: 'Hesperos', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ id: '69D4', source: 'Hespéros', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ id: '69D4', source: 'ヘスペロス', capture: false }),
+      response: Responses.spread(),
+    },
+    {
+      id: 'P4S Lava Pinax',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '69D5', source: 'Hesperos', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ id: '69D5', source: 'Hesperos', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ id: '69D5', source: 'Hespéros', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ id: '69D5', source: 'ヘスペロス', capture: false }),
+      infoText: (_data, _matches, output) => output.groups!(),
+      outputStrings: {
+        groups: {
+          en: 'Healer Groups',
+          de: 'Heiler-Gruppen',
+          fr: 'Healers en groupes',
+          ja: 'ヒラに頭割り',
+          cn: '治疗分摊组',
+          ko: '힐러 그룹 쉐어',
+        },
+      },
+    },
+    {
+      id: 'P4S Northerly Shift Slash',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '6A02', source: 'Hesperos', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ id: '6A02', source: 'Hesperos', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ id: '6A02', source: 'Hespéros', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ id: '6A02', source: 'ヘスペロス', capture: false }),
+      alertText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Go North Edge',
```

Comment:
North Cleave?

Reply:
I agree with this. Was copied from p4n, but this makes it easier to tell knockback vs cleave.